### DISCUSSION TITLE
Here's the daily graph. I want to change this to a cumulative graph.

### DISCUSSION THREAD

**Discussion by sohwaje** [grafana/grafana-infinity-datasource#766] [Q&A]
![스크린샷 2024-02-14 오후 9 00 07](https://github.com/grafana/grafana-infinity-datasource/assets/19942049/c40b1bfb-afbf-46d3-a19d-fc2c557ddfa4)

As above, the graph is drawn clearly with the daily, but I don't want this, I want the accumulated graph. Is there a way?

**Comment by zoltanbedi** [comment]
This is not related to the plugin. Please ask your questions in the [public slack](https://grafana.slack.com/) channel or in the [forum](https://community.grafana.com/).
